DO NOT MERGE: Updates to Cloud Docker Varnish container default config and install options (#6368)
magento/devdocs: src/cloud/docker/docker-containers.md

The `docker:build` command runs in interactive mode and verifies the configured service versions. To skip the interactive mode, use the `-n, --no-interaction` option.

> The **Varnish** service is installed by default, and when deployment completes Magento is configured to use Varnish for full page cacheing (FPC) for Magento version 2.2.0 or later. The configuration process preserves any custom FPC configurations that you have already configure.

Reviewer: ```suggestion
The **Varnish** service is installed by default, and when deployment completes Magento is configured to use Varnish for full page cacheing (FPC) for Magento version 2.2.0 or later. The configuration process preserves any custom FPC configuration settings that already exist.
```

The `docker:build` command runs in interactive mode and verifies the configured service versions. To skip the interactive mode, use the `-n, --no-interaction` option.

> **Varnish** service is installed by default and right after the deployment Magento is configured to use Varnish for FPC.

Reviewer: ```suggestion
**Varnish** service is installed by default and right after the deployment Magento is configured to use Varnish for full page cacheing (FPC).
```

> You may avoid installation of Varnish using `--no-varnish` option.

Reviewer: ```suggestion
You can skip installation of the Varnish container by adding the `--no-varnish` option to the `docker:build` command when you are creating a Cloud Docker environment.
```
